android/res/resources.properties

```diff
@@ -0,0 +1 @@
+unqualifiedResLocale=en-US
```

Comment:
Should it be en instead? Does it work with different languages?

Reply:
I chose en-US following the provided example here https://developer.android.com/build/releases/gradle-plugin#groovy, and I can change it based on your preference.
I haven't used different languages, but it is expected to work correctly.

Reply:
Please test all your changes on a real device.

Reply:
It's working with other languages. Can you please verify using your device?

Reply:
No, I can't. Did you test it with en-GB, en-AU, en-CA, etc.?

Reply:
Yes, I followed the steps in INSTALL.md for testing (using WSL)